docs/coins.md

```diff
@@ -45,7 +45,7 @@ This list is generated from [./coins.json](../coins.json)
 | 500     | Theta            | THETA  | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/theta/info/logo.png" width="32" />        | <https://www.thetatoken.org>  |
 | 501     | Solana           | SOL    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/solana/info/logo.png" width="32" />       | <https://solana.com>          |
 | 700     | xDai             | xDAI   | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/xdai/info/logo.png" width="32" />         | <https://poa.network/xdai>    |
-| 714     | Binance          | BNB    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/binance/info/logo.png" width="32" />      | <https://binance.org>         |
+| 714     | BNB              | BNB    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/binance/info/logo.png" width="32" />      | <https://binance.org>         |
```

Comment:
```suggestion
| 714     | BNB              | BNB    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/bnb/info/logo.png" width="32" />      | <https://binance.org>         |
```

Changes made here https://github.com/trustwallet/assets/pull/270

Reply:
damn, this needs to change codegen